examples/twitter_inlet.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+import tweepy
+from databay import Inlet, Link, Record
+from databay.outlets import PrintOutlet
+from databay.planners import ApsPlanner, SchedulePlanner
+from tweepy.models import Status
+
+
+class TwitterTimelineInlet(Inlet):
+
+    def __init__(self, api: tweepy.API, most_recent_id=None, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.api = api
+
+        # this will ensure we only every pull tweets that haven't been handled
+        self.most_recent_id = most_recent_id
```

Comment:
Very cool you have this functionality in this example. Someone just asked me for this in a recent Reddit thread.